utils/ccloud_library.sh

```diff
@@ -293,6 +293,17 @@ function ccloud::validate_schema_registry_up() {
   return 0
 }
 
+function ccloud::get_environment_id_from_service_id() {
+  SERVICE_ACCOUNT_ID=$1
+
+  ENVIRONMENT_NAME_PREFIX=${ENVIRONMENT_NAME_PREFIX:-"demo-env-$SERVICE_ACCOUNT_ID"}
+  ENVIRONMENT=$(ccloud environment list -o json | jq -r 'map(select(.name | startswith("'"$ENVIRONMENT_NAME_PREFIX"'"))) | .[].id')
```

Comment:
I don't think the intention of this function is to modify a global instance of `ENVIRONMENT`, but this will. I suggest we either echo the result of the previous command without assignment, or we use the `local` modifier (and maybe a different var name) for the variable to avoid mutating the global version of this.

Reply:
@rspurgeon thoughts on latest change?